index.bs

```diff
@@ -538,23 +546,23 @@ The <dfn attribute for=FileSystemEntry>filesystem</dfn> getter steps are to retu
 
 The <dfn method for=FileSystemEntry>getParent(|successCallback|, |errorCallback|)</dfn> method steps are:
 
-1. [=Queue a task=] to perform the following steps:
+1. [=In parallel=], run these steps:
```

Comment:
Is "In parallel" actually the right/desirable thing to do here, or should all these operations use a (shared) [parallel queue](https://html.spec.whatwg.org/multipage/infrastructure.html#parallel-queue) instead, similar to how the [file system spec](https://fs.spec.whatwg.org/#file-system-queue) does things? 
I guess with the read-only interface the entries-api defines there probably isn't much of a observable difference, as it shouldn't matter in what order things execute, so I guess plain in parallel might be good enough for this spec.

Reply:
Tracing through the "in parallel" steps, where this looks like it might matter is the `readEntries()` impl, which modifies state of the reader. It currently uses a "reading flag" to prevent multiple calls at once. At least as currently written, it looks like there could be a race, where the in-parallel steps "Clear this's reading flag" and then do a bunch of work that includes modifying state (e.g. "set this's done flag"). 

So yeah, this should be reworked further.

Reply:
`readEntries()` was reworked.